spec/lib/sidekiq/honeycomb_middleware_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+class TestSidekiqWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :low_priority
+
+  def perform(arg = nil)
+    raise StandardError, "BOOM" if arg == "fail"
+  end
+end
+
+RSpec.describe Sidekiq::HoneycombMiddleware do
+  let(:expected_hash) do
+    {
+      "sidekiq.class" => TestSidekiqWorker.to_s,
+      "sidekiq.queue" => "low_priority",
+      "sidekiq.jid" => instance_of(String),
+      "sidekiq.args" => ["dont fail"],
+      "sidekiq.result" => "success"
+    }
+  end
+
+  before do
+    Sidekiq::Testing.inline!
```

Comment:
This is dangerous because otherwise you set `.inline!` for all the other tests in the other files ran after this one. You should either use the block version of `.inline!` or call `.fake!` in an `after` block